.github/workflows/pr-tests.yml

```diff
@@ -0,0 +1,41 @@
+name: PR tests
+
+on:
+  pull_request:
+
+env:
+  CARGO_TERM_COLOR: always
+
+jobs:
+  test_debug:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v3
+      with:
+        submodules: recursive
+    - name: ⚡ Cache rust
+      uses: actions/cache@v3
+      with:
+        path: |
+          ~/.cargo/registry
+          ~/.cargo/git
+          target
+        key: ${{ runner.os }}-cargo-${{ hashFiles('**/Cargo.toml') }}
+    - name: ⚡ Cache nodejs
+      uses: actions/cache@v3
+      with:
+        path: |
+          ~/pilcom/node_modules
+        key: ${{ runner.os }}-pilcom-node-modules
+    - name: Install pilcom
+      run: git clone https://github.com/0xPolygonHermez/pilcom.git  && cd pilcom && npm install
+    - name: Build
+      run: cargo build
+    - name: Run tests
+      run: PILCOM=$(pwd)/pilcom/ cargo test --verbose
```

Comment:
I think we should run at least some ignored tests in the pr. Maybe keccak or something else end-to-end.

Reply:
I'll put keccak and vec-median, then. Lets see how long it takes.

Reply:
12 min. It is fine, I think.